src/utils/proptypes.js

```diff
@@ -60,6 +60,8 @@ export const rulerPropType = PropTypes.shape({
   visible: PropTypes.bool,
   xLabel: PropTypes.func.isRequired,
   yLabel: PropTypes.func.isRequired,
+  // a timestamp representing the initial position of the ruler
+  position: PropTypes.number,
```

Comment:
`position` is super vague -- what about `timestamp` or something?

Reply:
(and while I'm looking at this, I see that `xLabel` should be `timeLabel` . . .)